news/2025/2025-04-08-beatmap-spotlights-revival-spring-2025.md

```diff
@@ -0,0 +1,269 @@
+---
+layout: post
+title: "Beatmap Spotlights Revival: Spring 2025"
+date: 2025-04-08 23:00:00 +0000
+---
+
+Haven't heard this name for a while? Check out all the major changes to Beatmap Spotlights! Are you interested in helping out with the project? This time, we are also seeking fresh blood to ensure the project can run smoothly for many seasons to come!
+
+![](/wiki/shared/news/2025-04-08-beatmap-spotlights-revival-spring-2025/banner.jpg)
+
+It's been some time since we last met here indeed... let's examine what happened in the meantime, since there surely is a lot to cover.
+
+Even though the playlists are up and running already, we invite you to read on for the story about the new Beatmap Spotlights infrastructure, and the path that led us to this very day below!
+
+## A lengthy history lesson
+
+Heading into 2024, right after the conclusion of the Autumn 2023 season, the Beatmap Spotlights project was halted by the development team, due to an osu!(lazer) scoring rollout and infrastructure changes related to that.
+
+Some time passed, and alongside the [introduction of Daily Challenges](https://osu.ppy.sh/home/news/2024-07-25-introducing-daily-challenges) in July 2024, internal discussions arose about how the projects would coexist in the future. After some team-wide deliberations, it was decided that they would both focus on different matters, and aim for a varying player base.
+
+With that, the Beatmap Spotlights could run again. However, before that happens, various parts of Spotlights infrastructure would need to see some change.
+
+Back in September 2024 then, meetings were held and plans were drawn, and as a result the [Beatmap Spotlight Curators](https://osu.ppy.sh/groups/48) were met with a relatively simple announcement: the new goal was to stop using spreadsheets and scripts to run the Beatmap Spotlights, for sake of both smoother internal workflows, and enhanced player experience.
```

Comment:
```suggestion
Back in September 2024, meetings were held and plans were drawn, and as a result the [Beatmap Spotlight Curators](https://osu.ppy.sh/groups/48) were met with a relatively simple announcement: the new goal was to stop using spreadsheets and scripts to run the Beatmap Spotlights, for sake of both smoother internal workflows, and enhanced player experience.
```
I would just remove this entire section. "We had a ton of important meetings, and as a result we pinged people" what purpose does this serve?

Reply:
used to cover more meaning before Wala's rewordings so can indeed remove for now